entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [[ "${1#-}" != "$1" ]]; then
+    set -- manager "$@"
```

Comment:
What does this command do? Add a comment?

Reply:
has removed